src/wp-includes/cache-compat.php

```diff
@@ -35,3 +35,86 @@ function wp_cache_get_multiple( $keys, $group = '', $force = false ) {
 		return $values;
 	}
 endif;
+
+if ( ! function_exists( 'wp_cache_delete_multiple' ) ) :
+	/**
+	 * Retrieves multiple values from the cache in one call.
+	 *
+	 * Compat function to mimic wp_cache_delete_multiple().
+	 *
+	 * @ignore
+	 * @since 6.0.0
+	 *
+	 * @see wp_cache_delete_multiple()
+	 *
+	 * @param array  $keys  Array of keys under which the cache to deleted.
+	 * @param string $group Optional. Where the cache contents are grouped. Default empty.
+	 * @return array Array of return values organized into groups.
+	 */
+	function wp_cache_delete_multiple( $keys, $group = '' ) {
+		$values = array();
+
+		foreach ( $keys as $key ) {
+			$values[ $key ] = wp_cache_delete( $key, $group );
+		}
+
+		return $values;
+	}
+endif;
+
+
+if ( ! function_exists( 'wp_cache_add_multiple' ) ) :
+	/**
+	 * Add multiple values from the cache in one call.
+	 *
+	 * Compat function to mimic wp_cache_set_multiple().
+	 *
+	 * @ignore
+	 * @since 6.0.0
+	 *
+	 * @see wp_cache_add_multiple()
+	 *
+	 * @param array  $data  Array of key and value to be set.
+	 * @param string $group Optional. Where the cache contents are grouped. Default empty.
+	 * @param int    $expire Optional. When to expire the cache contents, in seconds.
+	 *                           Default 0 (no expiration).
+	 * @return array Array of return values organized into groups.
+	 */
+	function wp_cache_add_multiple( $data, $group = '', $expire = 0 ) {
+		$values = array();
+
+		foreach ( $data as $key => $value ) {
+			$values[ $key ] = wp_cache_add( $key, $value, $group, $expire );
+		}
+
+		return $values;
+	}
+endif;
+
+if ( ! function_exists( 'wp_cache_set_multiple' ) ) :
+	/**
+	 * Set multiple values from the cache in one call.
+	 *
+	 * Compat function to mimic wp_cache_set_multiple().
+	 *
+	 * @ignore
+	 * @since 6.0.0
+	 *
+	 * @see wp_cache_set_multiple()
+	 *
+	 * @param array  $data  Array of key and value to be set.
+	 * @param string $group Optional. Where the cache contents are grouped. Default empty.
+	 * @param int    $expire Optional. When to expire the cache contents, in seconds.
+	 *                           Default 0 (no expiration).
+	 * @return array Array of return values organized into groups.
+	 */
+	function wp_cache_set_multiple( $data, $group = '', $expire = 0 ) {
```

Comment:
Same as above.

```suggestion
	function wp_cache_set_multiple( array $data, $group = '', $expire = 0 ) {
```

Reply:
Actioned in https://github.com/WordPress/wordpress-develop/pull/2018